provider/repodata/data/json/currency/essential@1/es.json

```diff
@@ -0,0 +1,1209 @@
+{
+  "currency_patterns_map": {
+    "AED": {
+      "short_pattern_standard": true,
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 0,
```

Comment:
Issue: The place holder at index 0 is "AED" which is the ISO code. This is supposed to be filtered out. Please investigate.

Reply:
This is not an issue, the place holder is the same as the `ISO` code. if we decided to filter it out, this means what ?
This means that if the place holder equals the `ISO` code, we will give the index value of `u16::MAX`. and then, we could do not differentiate between the missing of a place holder or the place holder equals to the `iso` code.

Reply:
Why do we need that distinction?

We support formatting all currencies, even ones where we don't have data. If we don't have data, we use the ISO code (and presumably the alphaNextToNumber pattern). If that behavior is the same as what CLDR prescribes, then we don't need to carry the CLDR data.